### DISCUSSION TITLE
Reopen #27308 aka. add option to disable tracking scroll position

### DISCUSSION THREAD

**Discussion by Schweinepriester** [gatsbyjs/gatsby#34432] [Ideas / Feature Requests]
#27308 was closed due to inactivity, which imho isn't correct. I would have opened a new issue [as suggested](https://github.com/gatsbyjs/gatsby/issues/27308#issuecomment-1007919983), but because of the templates I was redirected here.

To quote from the [comment](https://github.com/gatsbyjs/gatsby/issues/27308#issuecomment-1007827065) I made there:

> I'd like to be able to disable [scroll restoration](https://www.gatsbyjs.com/docs/how-to/routing/scroll-restoration/)/tracking because of performance (for a feature we don't use):
> - https://css-tricks.com/a-primer-on-the-different-types-of-browser-storage/
> - https://web.dev/storage-for-the-web/
>
> [SessionStorage] … should be used with caution because it is synchronous and will block the main thread.

**Comment by jayjayjpg** [comment]
+1 and I'd also be up for working on it if this ticket was reopened

**Reply by jayjayjpg** [reply]
WIP: https://github.com/gatsbyjs/gatsby/pull/34530
